### PR TITLE
Pytest fixture for adaptive trapezoid numpy wrapper

### PR DESCRIPTION
**Pull request by kylemasc917**
## Contents
  Contains a pytest fixture for the adaptive trapezoid calculus function generated by @abruns123

### CODE REVIEW

**Comment by zbpetersbuf** [comment]
please do

File: test_calculus.py [hunk 1]
```diff
@@ -66,3 +66,35 @@ def test_trapezoid_scipy():
     Unit test for scipy implementation of trapezoid method
     '''
     assert np.isclose(calc.trapezoid_scipy(np.sin, 0, np.pi), 2)
+
+def d3(x):
```
Comment (iglesias-cardinale) [inline test_calculus.py:70]: I don't see the usefulness of these functions

File: test_calculus.py [hunk 2]
```diff
@@ -66,3 +66,35 @@ def test_trapezoid_scipy():
...
+        return np.sin(1 / x) / x**2
+
+@pytest.mark.parametrize("func, bounds, d, sens, expected", [
+    ("x^3+1", [0, 1], 100, 1, 1.25),  # Integral of x^3 + 1 from 0 to 1
+    ("exp(-1/x)", [1, 2], 100, 1, 0.5047),  # Approximation
+    ("cos(1/x)", [0.1, 0.2], 100, 1, 0.0322),  # Approximation
+])
+def test_adapt(func, bounds, d, sens, expected):
+    """
+    Unit test for adaptive integration function
```
Comment (iglesias-cardinale) [inline test_calculus.py:91]: A more descriptive docstring detailing what this test does would be nice

File: test_calculus.py (hunk 1 above)
Comment (iglesias-cardinale) [reply test_calculus.py:70]: I see that calc.adapt needs these to be defined, but they're alread defined in calculus.py

**Review by iglesias-cardinale — APPROVED** [review]
Code is well formed and tests/lints properly.

There could be a little more description in the docstrings, but it's not super necessary. 

I like that the unit test tests for all three functions. That's a clever way to do that. 

Approving for merge.

**Comment by kylemasc917** [comment]
@zbpetersbuf merge conflicts have been resolved. Should be good to go.

**Review by zbpetersbuf — APPROVED** [review]

**Pull request merged**